RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -2383,83 +1989,17 @@ namespace lst {
-    } else if (modulesInGPU.subdets[lowerModuleIndex1] == lst::Barrel and
-               modulesInGPU.subdets[lowerModuleIndex2] == lst::Barrel and
-               modulesInGPU.subdets[lowerModuleIndex3] == lst::Barrel and
-               modulesInGPU.subdets[lowerModuleIndex4] == lst::Endcap and
-               modulesInGPU.subdets[lowerModuleIndex5] == lst::Endcap) {
-      if (modulesInGPU.layers[lowerModuleIndex1] == 1) {
-        matchedRadii =
-            matchRadiiBBBEE12378(acc, innerRadius, bridgeRadius, outerRadius, bridgeRadiusMin2S, bridgeRadiusMax2S);
-      } else if (modulesInGPU.layers[lowerModuleIndex1] == 2) {
-        matchedRadii =
-            matchRadiiBBBEE23478(acc, innerRadius, bridgeRadius, outerRadius, bridgeRadiusMin2S, bridgeRadiusMax2S);
-      } else {
-        matchedRadii =
-            matchRadiiBBBEE34578(acc, innerRadius, bridgeRadius, outerRadius, bridgeRadiusMin2S, bridgeRadiusMax2S);
-      }
-    }
-
-    else if (modulesInGPU.subdets[lowerModuleIndex1] == lst::Barrel and
-             modulesInGPU.subdets[lowerModuleIndex2] == lst::Barrel and
-             modulesInGPU.subdets[lowerModuleIndex3] == lst::Endcap and
-             modulesInGPU.subdets[lowerModuleIndex4] == lst::Endcap and
-             modulesInGPU.subdets[lowerModuleIndex5] == lst::Endcap) {
-      matchedRadii = matchRadiiBBEEE(acc, innerRadius, bridgeRadius, outerRadius, bridgeRadiusMin2S, bridgeRadiusMax2S);
-    } else if (modulesInGPU.subdets[lowerModuleIndex1] == lst::Barrel and
-               modulesInGPU.subdets[lowerModuleIndex2] == lst::Endcap and
-               modulesInGPU.subdets[lowerModuleIndex3] == lst::Endcap and
-               modulesInGPU.subdets[lowerModuleIndex4] == lst::Endcap and
-               modulesInGPU.subdets[lowerModuleIndex5] == lst::Endcap) {
-      matchedRadii = matchRadiiBEEEE(acc,
-                                     innerRadius,
-                                     bridgeRadius,
-                                     outerRadius,
-                                     innerRadiusMin2S,
-                                     innerRadiusMax2S,
-                                     bridgeRadiusMin2S,
-                                     bridgeRadiusMax2S);
-    } else {
-      matchedRadii = matchRadiiEEEEE(acc,
-                                     innerRadius,
-                                     bridgeRadius,
-                                     outerRadius,
-                                     innerRadiusMin2S,
-                                     innerRadiusMax2S,
-                                     bridgeRadiusMin2S,
-                                     bridgeRadiusMax2S);
-    }
 
-    //compute regression radius right here - this computation is expensive!!!
-    if (not matchedRadii)
-      return false;
+    // 5 categories for sigmas
+    float sigmas2[5], delta1[5], delta2[5], slopes[5];
+    bool isFlat[5];
```

Comment:
is it 5->`Params_T5::kLayers` here? or is it some different 5?
perhaps xVec, yVec can be explicitly sized as well